arcgis-ios-sdk-samples/Features/Generate geodatabase/GenerateGeodatabaseViewController.swift

```diff
@@ -108,10 +108,10 @@ class GenerateGeodatabaseViewController: UIViewController {
                 // kick off the job
                 generateJob.start(
                     statusHandler: { (status: AGSJobStatus) in
-                        SVProgressHUD.show(withStatus: status.statusString())
+                        UIApplication.shared.showProgressHUD(status.statusString())
                     },
                     completion: { [weak self] (object: AnyObject?, error: Error?) in
```

Comment:
Not related to the work in this PR, but happened to notice that `object: AnyObject?` should be `geodatabase: AGSGeodatabase?`. Could you please create an issue and/or submit a PR to fix that?

Reply:
👍

Reply:
PR at #1060